docs/get-started/sumo-logic-ui.md

```diff
@@ -1,115 +1,143 @@
+To manage data in Sumo Logic, do the following:
 
-![WTS_UI_LogSearch-Metrics-LiveTail.png](/img/get-started/ui/WTS_UI_LogSearch-Metrics-LiveTail.png)
+1. From the top-right navigation bar, click **Configuration**.<br/><img src={useBaseUrl('img/get-started/config.png')} alt="img/get-started/config.png" width="400"/>
+1. Choose from the following, as needed:
+    * **Collection**. [Manage Collectors and Sources](/docs/send-data/collection).
+    * **Logs**. Manage [fields](/docs/manage/fields), [field extraction rules](/docs/manage/field-extractions), [partitions](/docs/manage/partitions-data-tiers), [scheduled views](/docs/manage/scheduled-views) <!--, [connections](/docs/manage/connections-integrations), and [data forwarding](/docs/manage/data-forwarding). -->
+    * **Metrics**. Manage metrics rules, [logs-to-metrics](../metrics/logs-to-metrics.md), and [metrics transformation rules](../metrics/metrics-transformation-rules.md).
+    * **Cloud SIEM Integrations**.
+    * **Cloud SIEM Entities**.
+    * **Cloud SIEM Workflow**.
 
-To access additional Log Search, Metrics, and Live Tail options, do the following:
+### Manage accounts, users, and security
 
-1. Hover the cursor over the tab **Details** icon.
+Sumo Logic administrators (admins) manage user accounts, user roles, and security. You must have Sumo Logic Admin role privileges to perform these tasks.
 
-    ![WTS_Tab-options-details.png](/img/get-started/ui/WTS_Tab-options-details.png)
+To administer Sumo Logic accounts, users, and security, do the following:
 
-1. Select the tab and choose an option from the dropdown menu.
+1. From the top navigation bar and click **Administration**.<br/> <img src={useBaseUrl('img/get-started/admin.png')} alt="img/get-started/admin.png" width="400"/>
+1. Choose from the following, as needed: <!-- what does this look like for Admin users? -->
+    * **Account**. [View information about your organization's Sumo Logic subscription](/docs/manage/manage-subscription), [enable and manage the data volume index,](/docs/manage/ingestion-volume/data-volume-index) [manage billing](/docs/manage/manage-subscription).
+    * **Users and Roles**. [Manage users and roles](/docs/manage/users-roles).
+    * **Security**. [Set password policy for your org](/docs/manage/security/set-password-policy), [set up security whitelist](/docs/manage/security/create-allowlist-ip-cidr-addresses), [manage access keys](/docs/manage/security/access-keys), manage security policies ([audit index](/docs/manage/security/audit-indexes/audit-index)), [support account access](/docs/manage/security/enable-support-account), [dashboard sharing](../dashboards/share-dashboard-new), and [set up SAML authentication](/docs/manage/security/saml).
```

Comment:
@agaur Can you share what options appear beneath **Administration**? I don't have Admin access, so I'm seeing limited options:
<img width="300" alt="Screenshot 2023-12-26 at 4 25 17 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/cfced5c5-75a2-4782-b4f3-38e37f0c0684">

Reply:
@kimsauce, here is what I see:
<img width="273" alt="Screenshot 2024-01-03 at 3 13 16 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/127961815/7c1a1c09-b602-487c-b8c2-c81079138d0d">

Reply:
@jpipkin1 , oh wow, that's totally different. It looks like you're on stag and I'm on cust. usage. @agaur can you clarify?

Reply:
Hi @kimsauce, 
Yes, the items that you see in the admin menu will be based on your permissions. This behaviour has not changed from Bento. Whatever administration pages the user had access to in Bento, they will be accessible in Kanso as well. That's why the different list on stagData and customerUsage because user permissions are different.